doc/contributing/docs/examples.rst

```diff
@@ -1,72 +1,112 @@
-        ---
-        ...
-        tarantool> my_fiber_object = my_fiber.create(function_name)
-        ---
-        ...
+        :param function: the function to be associated with the fiber
+        :param function-arguments: what will be passed to function
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-               Module, class and method
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        :return: created fiber object
+        :rtype: userdata
 
-Here is an example of documenting a module (``my_box.index``), a class
-(``my_index_object``) and a function (``my_index_object.rename``).
+        **Example:**
 
-.. module:: my_box.index
+        ..  code-block:: tarantoolsession
 
-.. class:: my_index_object
+            tarantool> fiber = require('fiber')
+            ---
+            ...
+            tarantool> function function_name()
+                     >   print("I'm a fiber")
+                     > end
+            ---
+            ...
+            tarantool> fiber_object = fiber.create(function_name); print("Fiber started")
+            I'm a fiber
+            Fiber started
+            ---
+            ...
 
-    .. method:: rename(index-name)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Documenting class method and data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-        Rename an index.
+Here is an example of documenting a method and a data of a class ``index_object``:
 
-        :param index_object: an object reference
-        :param index_name: a new name for the index (type = string)
+..  code-block:: rst
 
-        :return: nil
+    ..  class:: index_object
 
-        Possible errors: index_object does not exist.
+        ..  method:: get(key)
 
-        **Example:**
+            Search for a tuple via the given index, as described :ref:`here <box_index-note>`.
 
-        .. code-block:: tarantoolsession
+            :param index_object index_object: an :ref:`object reference
+                                              <app_server-object_reference>`.
+            :param scalar/table      key: values to be matched against the index key
 
-            tarantool> box.space.space55.index.primary:rename('secondary')
-            ---
-            ...
+            :return: the tuple whose index-key fields are equal to the passed key values
+            :rtype:  tuple
+
+            **Possible errors:**
+
+            * no such index;
+            * wrong type;
+            * more than one tuple matches.
+
+            **Complexity factors:** Index size, Index type.
+            See also :ref:`space_object:get() <box_space-get>`.
+
+            **Example:**
+
+            ..  code-block:: tarantoolsession
+
+                tarantool> box.space.tester.index.primary:get(2)
+                ---
+                - [2, 'Music']
+                ...
+
+        For documenting a data, a description, a return type and an example will be enough:
```

Comment:
again, just data

Reply:
Also, for documenting ____ what? It needs an object, otherwise the reader can understand it so that data *is* the object of documenting.

Reply:
a return type, and an example – Oxford comma